runtime.txt

```diff
@@ -1 +1 @@
-python-2.7.13
\ No newline at end of file
+python-3.6.2
```

Comment:
Is this correct?

Reply:
@andela-ookoro will add this changes in his PR that upgrades to python3